mw-webapp/src/component/title/Title.module.scss

```diff
@@ -0,0 +1,7 @@
+@import "src/styles/variables.scss";
+
+.title {
+    padding-top: $title-padding;
+    color: $title;
```

Comment:
1. rename. not clear naming. it is hard to understand just by name that it is a color
2. think about architecture, you don't want to handle 100500 variable just in styles, right?

Reply:
Really I don't want to use a lot of variables. But at the moment, the color and padding variables are used (which is used in header). If there were other immutable properties, I would have already used them in title.module.scss)

Reply:
I mean naming, maybe we should use just a little bit more abstract name for color to make them reusable. For example textColor, or primaryTextColor... It is up to you